conduit/pipeline/pipeline.go

```diff
@@ -28,6 +28,30 @@ import (
 	"github.com/algorand/conduit/conduit/plugins/processors"
 )
 
+// ErrOverrideConflict is used when there are two different round overrides detected.
+type ErrOverrideConflict struct {
+	pluginOrCLI uint64
+	other       uint64
+	cli         bool
+}
+
+// makeErrOverrideConflict creates a new ErrOverrideConflict.
+func makeErrOverrideConflict(pluginOrCLI, other uint64, cli bool) error {
+	return ErrOverrideConflict{
+		pluginOrCLI: pluginOrCLI,
+		other:       other,
+		cli:         cli,
+	}
+}
+
+// Error implements the error interface.
+func (e ErrOverrideConflict) Error() string {
+	if e.cli {
+		return fmt.Sprintf("inconsistent round overrides detected: command line (%d), plugins (%d)", e.pluginOrCLI, e.other)
```

Comment:
This seems oddly named. The field is `pluginOrCLI` but the error message just says `command line`.

Reply:
The cli field is supposed to indicate which one it is. Maybe these inputs can be strings instead of numbers, then it can even identify which plugin provided the conflicting round. :thinking:

Reply:
Updated, this should be less convoluted now.